tests/test_message.py

```diff
@@ -38,6 +38,7 @@ def test_when_provided_a_full_message_it_parses_it_into_objects(self):
         self.assertEqual(message.sources[0].variables[0].relatedUrls[0].url, 'http://example.com/file649.txt')
         self.assertEqual(message.sources[0].variables[0].relatedUrls[0].mimeType, 'text/plain')
         self.assertEqual(message.sources[0].variables[0].relatedUrls[0].format, 'ASCII')
+        self.assertEqual(message.sources[0].variables[0].visualizations[0]['Name'], 'Test1234')
```

Comment:
We should test the top sources[0].visualizations at the collection level in addition to the variables level.

Reply:
Done